libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/DBOpenHelper.java

```diff
@@ -342,6 +342,33 @@ public static synchronized void deleteAllDatabases(Context ctx, UserAccount user
 		}
 	}
 
+	/**
+	 * One time upgrade steps from older versions to Mobile SDK 8.2. Only for internal use!
+	 *
+	 * @deprecated Will be removed in Mobile SDK 10.0.
+	 */
+	public static void upgradeTo8Dot2() {
```

Comment:
It's important that this lives inside `DBOpenHelper` and be executed from within here - otherwise the call is too late, i.e. the call can't be after `getWritableDatabase()` since that will result is attempting to open it with the wrong key.